Name: Change-Number-of-Bookmarks
Version: 1.3.1-1
Author: jmartino2011
Description: Change the number of bookmarks that show on the start page of the browser, instead of just the first 12 bookmarks you have

--- .orig/usr/palm/applications/com.palm.app.browser/app/controllers/startpage-assistant.js
+++ /usr/palm/applications/com.palm.app.browser/app/controllers/startpage-assistant.js
@@ -7,7 +7,7 @@
 	this._destinationUrl = params.lastUrl;
 	this._orientation = params.orientation || 'up';
 	this._bookmarks = [];
-	this._maxShowCount = 12;
+	this._maxShowCount = 100;
 	this.store = new BookmarkStore({
 		database: Mojo.Controller.appController.assistant.getDatabase()
 	});
